lib/node_modules/@stdlib/array/full/docs/types/index.d.ts

```diff
@@ -226,7 +53,7 @@ declare function full( length: number, value: any, dtype: 'generic' ): Array<any
 * var arr = full( 2, 1.0, 'float32' );
 * // returns <Float32Array>[ 1.0, 1.0 ]
 */
-declare function full( length: number, value: any, dtype?: DataType ): AnyArray;
+declare function full<T, U extends keyof DataTypeMap<number> = 'float64'>( length: number, value: T, dtype?: U ): DataTypeMap<T>[U];
```

Comment:
We don't need to do this in this PR, but we should consider making `'float64'` be an imported type literal, so we don't have to change the default in multiple places should the default ever change. Recently, I've been trying to reduce duplication by consolidating to `@stdlib/array/defaults` and `@stdlib/ndarray/defaults`.